packages/backend-modules/discussions/graphql/resolvers/Discussion/userCanComment.js

```diff
@@ -1,35 +1,33 @@
 const { Roles } = require('@orbiting/backend-modules-auth')
 const { hasUserActiveMembership } = require('@orbiting/backend-modules-utils')
 
-const DEFAULT_ROLES = ['member']
-
+/**
+ * User can comment when either:
+ * - a) it has a role which is allowed to comment (discussion.allowedRoles)
```

Comment:
being very pedantic here, but I think we shouldn't call user "it"
```suggestion
 * - a) they have a role which is allowed to comment (discussion.allowedRoles)
```